packages/client/src/runtime/core/types/exported/itxClientDenyList.ts

```diff
@@ -1,4 +1,4 @@
-const denylist = ['$connect', '$disconnect', '$on', '$transaction', '$use', '$extends'] as const
+const denylist = ['$connect', '$disconnect', '$on', '$use', '$extends'] as const
```

Comment:
Shouldn't this depend on the provider now?

Reply:
Yes correct, this is really only available for SQL servers. Do you know if there is a utility for determining if the provider is an SQL server?